packages/vscode-extension/WHATISNEW.md

```diff
@@ -19,6 +20,22 @@
 
 # Changelog
 
+## 4.0.4 - Aug 08, 2022
+
+Incremental version for Teams Toolkit with multiple bugs fix, enhancements and new additions to sample application gallery.
+
+New samples:
+- Proactive Messaging: Save users' conversation reference to send proactive reminder messages using bots.
+
+    ![Proactive Messaging](img/proactive_messaging.png) 
```

Comment:
I have a concern about the reference style in relative path of this png image, that is, it may not working after publish. I remember some related issue occurred a long time ago. Just to call out it.

Reply:
Thanks, updated to use remote file url now. URL should be available once merged.